### PR TITLE
Upgrade ASIO4ALL

### LINKED ISSUE #995: Upgrade ASIO4ALL to v2.16
**URL**

https://asio4all.org/version-2-16-released/

A new version of ASIO4ALL has been released. We should update the website. Reading the comments it seems that anti virus software flags it, so we may want to check if it's really safe before setting the link.

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**
Updates the ASIO4ALL link.

**Context: Fixes an issue? Related issues**
Fixes: #995 
**Status of this Pull Request**
Ready for review

**What is missing until this pull request can be merged?**
Nothing

**Does this need translation?**

YES

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

File: wiki/en/Installation-for-Windows.md [hunk 1]
```diff
@@ -35,11 +35,7 @@ Jamulus can use [ASIO](https://en.wikipedia.org/wiki/Audio_Stream_Input/Output).
 * If you have an external sound card/audio interface, use its official ASIO driver if you can (they usually provide the best quality).
 * If you don't have an external sound card, you will probably not have an ASIO driver so will need to install a generic one like ASIO4ALL:
 
-You can try two versions of ASIO4ALL. ASIO4ALL v2.14 includes a workaround for a bug which might break other functionality.
-
-[ASIO4ALL v2.15 Download](https://github.com/jamulussoftware/assets/raw/main/ASIO4ALL/v2.15/ASIO4ALL_2_15_English.exe){: .button target="_blank" rel="noopener noreferrer"}
-
-[ASIO4ALL v2.14 Download](https://github.com/jamulussoftware/assets/raw/main/ASIO4ALL/v2.14/ASIO4ALL_2_14_English.exe){: .button target="_blank" rel="noopener noreferrer"}
```
Comment (ann0see) [inline wiki/en/Installation-for-Windows.md:42]: No longer necessary.

Comment (ann0see) [reply wiki/en/Installation-for-Windows.md:42]: I can't test this really as my machine "works".

**Comment by pljones** [comment]
http://jamuluswebsite.drealm.info/wiki/Installation-for-Windows
Looks okay, click through popped up a download but I didn't try download/install.

**Review by pljones — APPROVED** [review]

**Comment by pljones** [comment]
Just to confirm I _did_ later try (on a 2.14 install) using the download link and it was problem (and pop-up ad) free.

**Pull request merged**